support-e2e/tests/utils/testUserDetails.ts

```diff
@@ -1,29 +1,40 @@
 import { Page } from '@playwright/test';
 import { TestFields } from './userFields';
 
+export const setTestUserConfirmRequiredDetails = async (
```

Comment:
Do we need this new function `setTestUserConfirmRequiredDetails`? Could we not just edit `setTestUserRequiredDetails` to accept a `confirmEmail` boolean and to include a conditional like:

```
if (confirmEmail) {
		await page.getByLabel('Confirm email address', { exact: true }).fill(email);
}
```

We could remove the 2 new functions `setTestUserConfirmRequiredDetails` and `setTestConfirmEmail` by doing this?

Reply:
@GHaberis : Have removed the extra function, renamed existing and used an optional `confirmEmail` boolean parameter now on this [PR here now.](https://github.com/guardian/support-frontend/pull/6858)